### ISSUE TITLE
关于将mits用于其它领域的请教

### ISSUE THREAD

**Issue by YangBo0411** [yoxu515/MITS#7]
作者您好，对您的研究工作很感兴趣，想将mits用于其它领域。目前只有一个数据集，该数据集包括box和mask两种标注样式，数据集形式如下，我应该如何调整我的数据集才能使用mits？
└───dataset
                              ‎‎‎‎‎‎‎‎‎                       ‎                     └───images
          ‎                                           ‎                     └───masks
         ‎‎‎‎‎‎‎‎‎‎‎‎‎‎‎                                            ‎                     └───boxes

**Comment by yoxu515** [comment]
你可以参考dataloaders/train_datasets.py中LaSOT或者GOT10K的实现，对照它们的格式来组织数据集，必要时可以自定义一个新的类来适配你的数据集
